src/IOUService/web/index.html

```diff
@@ -40,21 +40,19 @@ <h3 class="col-md-12">Send a <em>thank you</em> note!</h3>
             </div>
             
             <h3>Notes sent to me</h3>
-            <button type="button" class="btn btn-primary">
+            <button type="button" class="btn btn-primary" id="refresh_button">
```

Comment:
Consistency: the Send button specifies the type, id, class in this order. This other button also follow this order.